src/spike/defaultOpts.js

```diff
@@ -1,8 +1,13 @@
+'use strict'
+
 const extractors = require('./extractors')
 const transformers = require('./transformers')
```

Comment:
Shouldn't the extractors be moved into the versioned folder structure as well? `extractors.js` exports an object keyed with the IEFT versioned field names. i.e. `'(created)'`

Maybe the transformers should be versioned too? There don't seem to be version-specific functions yet but that might change later. Maybe this can move if and when there is a need. 

The default `signatureFields` below on lines 12-15 are also version-specific.

Reply:
Great call on the extractors. Maybe we should just move all defaults into the versions directory. That also makes more  sense for moving the tests across and having them run for each version.

> Would they need to require the older version defaults and pass them into the creator function to override the latest defaults?

☝️ Yup, this exactly. Require and use what you need. Thats why we allow the module to be fully configurable.